roles/acm_setup/README.md

```diff
@@ -24,6 +24,7 @@ The configuration of the ACM hub can be customized by using the following variab
 |hub_db_volume_size                  |40Gi                           |No           | This value specifies how much storage it is allocated for storing files like database tables and database views for the clusters. You might need to use a higher value if there are many clusters
 |hub_fs_volume_size                  |50Gi                           |No           | This value specifies how much storage is allocated for storing logs, manifests, and kubeconfig files for the clusters. You might need to use a higher value if there are many clusters
 |hub_img_volume_size                 |40Gi                           |No           | This value specifies how much storage is allocated for the images of the clusters. You need to allow 1 GB of image storage for each instance of Red Hat Enterprise Linux CoreOS
+|hub_image_service_skip_tls_verify   |false                          |No           | Skip TLS verification in the AgentServiceConfig for image service operations. Useful in environments with self-signed certificates or certificate issues.
```

Comment:
I would recommend naming the variable as `hub_img_svc_skip_tls_verify ` to follow the same naming pattern as hub_img_volume_size. 
Other than that, this looks good to me.